src/model.js

```diff
@@ -20,7 +20,12 @@ export default class Model {
         .replace(/^insert/i, 'INSERT OR IGNORE')
     )
 
-    return helpers.runQuery(this.ctx, query)
+    return helpers.runQuery({
+      query,
+      instance: this.ctx,
+      name: this.name,
```

Comment:
Per my other comment, knex unfortunately is not consistent about this so it's not there for inserts...